analyzers/tests/SonarAnalyzer.UnitTest/Rules/CalculationsShouldNotOverflowTest.cs

```diff
@@ -0,0 +1,57 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2023 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using SonarAnalyzer.Common;
+using SonarAnalyzer.Rules.CSharp;
+
+namespace SonarAnalyzer.UnitTest.Rules;
+
+[TestClass]
+public class CalculationsShouldNotOverflowTest
+{
+    private readonly VerifierBuilder builder = new VerifierBuilder()
+        .AddAnalyzer(() => new SymbolicExecutionRunner(AnalyzerConfiguration.AlwaysEnabled))
+        .WithBasePath(@"SymbolicExecution\Roslyn")
+        .WithOnlyDiagnostics(CalculationsShouldNotOverflow.S3949);
+
+    [TestMethod]
+    public void CalculationsShouldNotOverflow_CS() =>
+        builder.AddPaths("CalculationsShouldNotOverflow.cs").Verify();
+
+    [TestMethod]
+    public void CalculationsShouldNotOverflow_CS_CSharp8() =>
```

Comment:
`CSharp8` already contains the `CS` information :)
Also in the UTs below
```suggestion
    public void CalculationsShouldNotOverflow_CSharp8() =>
```